packages/validation/src/loadbalancers.schema.ts

```diff
@@ -155,36 +155,42 @@ export const UpdateRouteSchema = object({
 });
 
 export const UpdateConfigurationSchema = object({
-  label: string().min(1),
-  port: number(),
+  label: string().min(1, 'Label must not be empty.'),
+  port: number()
+    .min(1, 'Port must be greater than 0.')
+    .max(65535, 'Port must be less than 65535.')
```

Comment:
Sorry, I should have caught this when I caught the `min`:

Should `port` be _less than or equal to_ 65535 or _less than_ 65535?

If it's less than, then we should have 
```suggestion
    .max(65534, 'Port must be less than 65535.')
```
across the board for this field validation.